src/utils/spec/utils.spec.ts

```diff
@@ -3,12 +3,15 @@ import {
   parseLogLines,
   millisecondsToDdHhMmSs,
   inExistingProject,
-  diff
+  diff,
+  setupGitIgnore
 } from '../utils'
-import { createFile, deleteFile } from '../file'
+import { createFile, deleteFile, fileExists, readFile } from '../file'
 import path from 'path'
 
 describe('utils', () => {
+  const folderPath = 'src/utils/spec'
```

Comment:
Best to use `path.join` rather than hardcoded forward slashes here.

Reply:
good catch!

Reply:
fixed